app/scripts/migrations/120.2.test.ts

```diff
@@ -0,0 +1,576 @@
+import { cloneDeep } from 'lodash';
```

Comment:
Probably fine but would it be worth having a describe block where we mix in all of the expected state changes?

Reply:
Ah darn, we have one but I forgot to update it. I'll follow-up on `develop` to fix that, good catch

Reply:
Good idea. I just realized I forgot to update the JSDoc block for the overall migration as well. I'll follow up on `develop` with both improvements.

Reply:
Addressed here: https://github.com/MetaMask/metamask-extension/pull/26333